src/create-wa-contact/createWaContact.js

```diff
@@ -0,0 +1,111 @@
+// Assert [REQUIREMENTS] variables.
+
+console.log('\n__________________WA_____________________\n');
+console.log(JSON.stringify(run('whatsapp', 'ptBr')));
+console.log('\n');
+console.log(JSON.stringify(run('whatsapp', 'enUs')));
+console.log('\n__________________BC_____________________\n');
+console.log(JSON.stringify(run('blipchat', 'ptBr')));
+console.log('\n');
+console.log(JSON.stringify(run('blipchat', 'enUs')));
+console.log('\n_______________________________________\n');
+
+// Code of builder here
+
+function run(userChannel, userLanguage = 'ptBr') {
+    return getWaContact(userChannel, userLanguage);
+}
+
+function getWaContact(userChannel, userLanguage = 'ptBr') {
+    const contactName = {
+        ptBr: 'Nome do Contato',
+        enUs: 'Contact'
+    };
+
+    const phoneNumber = '55123456789';
+    const description = {
+        ptBr: 'Maçã',
+        enUs: 'Apple'
+    };
+
+    const props = {
+        userChannel,
+        contactName,
+        phoneNumber,
+        description,
+        userLanguage
+    };
+
+    return createWaContact(props);
+}
+
+// Below are all scripts used to create WA contact
+// It should be put in the router resources in order to be used by the above script
+
+function createWaContact({
+    userChannel,
+    contactName,
+    phoneNumber,
+    description = null,
+    userLanguage = 'ptBr'
+    } = {}) {
+    let contactWebLink = {};
+
+    if (description instanceof Object) {
+        description = description[userLanguage];
+    }
+
+    if (contactName instanceof Object) {
+        contactName = contactName[userLanguage];
+    }
+
+    if (userChannel === 'whatsapp') {
+        contactWebLink.type = 'application/json';
```

Comment:
Just a suggestion: why don't you put the build logic for each channel in separate functions? Like:
```
if (channel == 'someChannel') {
    channelWebLink = buildForSomeChannel(props);
} else {
    channelWebLink = buildForAnotherChannel(props);
}
return channelWebLink;
```